drivers/hwinfo/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
 zephyr_sources_ifdef(CONFIG_USERSPACE   hwinfo_handlers.c)
```

Comment:
typo in commit body, s/sport/support/

Reply:
It's used as a transitive verb, meaning "boast".